NorthstarDLL/scripts/scriptjson.cpp

```diff
@@ -212,10 +212,12 @@ ADD_SQFUNC(
 			doc.GetErrorOffset());
 
 		if (bFatalParseErrors)
+		{
 			g_pSquirrel<context>->raiseerror(sqvm, sErrorString.c_str());
```

Comment:
```suggestion
			return g_pSquirrel<context>->raiseerror(sqvm, sErrorString.c_str());
```

Reply:
Not adding this because it isn't consistent with the rest of the repository.

Reply:
L